.gitattributes

```diff
@@ -0,0 +1,18 @@
+/tests export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/phpcs.ruleset.xml  export-ignore
+/phpunit.xml.dist export-ignore
+/ISSUE_TEMPLATE.md export-ignore
+/PULL_REQUEST_TEMPLATE.md export-ignore
+#/changelog.md export-ignore
+#/readme.md export-ignore
+/vendor export-ignore
```

Comment:
so, at the moment, if you were to download the Zip file from the Github releases, the `vendor` directory _is_ included, as we want folks to be able to download the Zip, install in WordPress and be up and running. 

So we still need to include the `vendor` directory, at least for the time being